node/src/main.rs

```diff
@@ -18,6 +18,10 @@
 
 //! Darwinia node CLI.
 
+#![cfg_attr(
```

Comment:
Why allow the unused crate in those features?

Reply:
Perhaps you mean `disallow`? Enable `unused_crate_dependencies` in both `runtime-benchmarks` and `try-runtime` features. otherwise, the following errors will be detected:

```rs
error: external crate `frame_benchmarking` unused in `darwinia`: remove the dependency or add `use frame_benchmarking as _;`
  --> node/src/main.rs:19:1
   |
19 | //! Darwinia node CLI.
   | ^
   |
note: the lint level is defined here
  --> node/src/main.rs:26:9
   |
26 | #![deny(unused_crate_dependencies)]
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^

error: external crate `polkadot_parachain` unused in `darwinia`: remove the dependency or add `use polkadot_parachain as _;`
  --> node/src/main.rs:19:1
   |
19 | //! Darwinia node CLI.
   | ^

error: external crate `sc_keystore` unused in `darwinia`: remove the dependency or add `use sc_keystore as _;`
  --> node/src/main.rs:19:1
   |
19 | //! Darwinia node CLI.
   | ^

error: external crate `sp_inherents` unused in `darwinia`: remove the dependency or add `use sp_inherents as _;`
  --> node/src/main.rs:19:1
   |
19 | //! Darwinia node CLI.
   | ^

error: external crate `sp_io` unused in `darwinia`: remove the dependency or add `use sp_io as _;`
  --> node/src/main.rs:19:1
   |
19 | //! Darwinia node CLI.
   | ^

error: could not compile `darwinia` due to 5 previous errors
```
Some crates are used under certain gateway.